src/components/Footer.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { Filter } from '../types/Filter';
+
+const FILTER_LINKS: Filter[] = ['All', 'Active', 'Completed'];
```

Comment:
Use enum and `Object.values(your created enum)` for creating array with these values